tests/test_example.yaml

```diff
@@ -1,76 +1,26 @@
-# person:
-#   # Test more complex nested structures
-#   education:
-#     - degree: BSc
-#       major: Computer Science
-#       year: 2018
-#       grades:
-#         math: 95.5
-#         programming: 98.2
-#     - degree: MSc
-#       major: Data Science
-#       year: 2020
-#       thesis:
-#         title: ML Applications
-#         advisor: Dr. Smith
-
-#   # Test arrays with mixed types
-#   contact_info:
-#     - type: email
-#       value: jane@example.com
-#       primary: true
-#     - type: phone
-#       value: 555-0123
-#       primary: false
-
-#   # Test deeper nesting levels
-#   projects:
-#     current:
-#       - name: Project A
-#         details:
-#           priority: high
-#           deadline: 2023-12-31
-#           tasks:
-#             - task1
-#             - task2
-#         team:
-#           - role: lead
-#             member: Jane
-#           - role: dev
-#             member: Bob
-#     completed:
-#       - name: Project B
-#         details:
-#           date: 2023-06-30
-#           outcome: success
-
-#   # Test various data formats
-#   data_types:
-#     integers:
-#       positive: 42
-#       negative: -17
-#       zero: 0
-#     floats:
-#       pi: 3.14159
-#       e: 2.71828
-#       negative: -0.5
-#     booleans:
-#       true_val: true
-#       false_val: false
-#     strings:
-#       empty: ""
-#       spaces: "hello world"
-#       special: "Line 1\nLine 2"
-
 company:
   name: Example Corp
   founded: 2001
   employees: 150
-  # departments:
-  #   - HR
-  #   - Engineering
-  #   - Marketing
+  flow_sequence: [1, 2, 3]
+  flow_sequence_float: [1.1, 2.2, 3.3]
+  flow_sequence_string: ["one", "two", "three"]
+  departments:
+    - HR
+    - Engineering
+    - Marketing
```

Comment:
Looks like test needs to be updated to test these are loaded properly

Reply:
Started towards this in 2290ef9

Reply:
Now have the first case here working, but as strings, since `%sequence` is strings. There isn't a `%sequence_int` or such yet.